adaptavist/adaptavist.py

```diff
@@ -887,6 +887,24 @@ def add_test_result_attachment(
             else self._upload_file(request_url, attachment, filename)
         )
 
+    def add_test_run_attachment(
+        self, test_run_key: str, attachment: Union[str, BinaryIO], filename: str = ""
+    ) -> bool:
```

Comment:
I ran the workflows on my local machine. Please change this line to this.

```suggestion
        self, test_run_key: str, attachment: Union[str, BinaryIO], filename: str = "" ) -> bool:
```